api/client/utils.go

```diff
@@ -138,3 +141,28 @@ func (cli *DockerCli) getTtySize() (int, int) {
 	}
 	return int(ws.Height), int(ws.Width)
 }
+
+func saveResponeBody(responseBody *io.ReadCloser, outfile *string) error {
```

Comment:
Could you make this func:

```
func copyToFile(outfile string, rc io.ReadCloser) error
```

So that there are no unneeded pointers and the parameters order is the same as for `io.Copy` (as they are being used next to each other).